src/Relude/Extra/CallStack.hs

```diff
@@ -10,8 +10,9 @@ import Relude
 import GHC.Stack (HasCallStack, callStack, getCallStack)
 
 -- TODO: better name?
-{- | This function returns name of the caller for this function. But it requires
-for caller to have 'HasCallStack' constraint. Otherwise it returns @"ownName".
+{- | This function returns the name of its caller function, but its requires
```

Comment:
Sorry, I made a typo in the first review, `but it requires`, `it` instead of `its`.